go.mod

```
@@ -1,4 +1,4 @@
-module github.com/aptible/go-deploy
+module github.com/reggregory/go-deploy
```

```suggestion
module github.com/aptible/go-deploy
```
Let's flip this back